modules/ROOT/pages/installation/linux/tarball.adoc

```diff
@@ -42,6 +42,13 @@ chown -R neo4j:adm /opt/neo4j-enterprise-{neo4j-version-exact}
 ----
 chown -R neo4j /opt/neo4j-enterprise-{neo4j-version-exact}
 ----
+. xref:configuration/file-locations.adoc#file-locations-file-locations[Change the default locations] of the _data_, _conf_, _certificates_, _licenses_, and _plugins_ (if you plan to use custom plugins) directories by setting the environment variable `NEO4J_CONF` and the respective xref:configuration/configuration-settings.adoc#_server_directories_settings[`server.directories.*`] settings to point to the desired locations.
++
+[NOTE]
+====
+Storing your Neo4j files outside `NEO4J_HOME` will simplify the upgrade process later because you will be able to replace the DBMS binaries without affecting the configuration and state.
+Otherwise, these Neo4j files will remain in the old installation folder and may be accidentally overwritten during an upgrade or deleted during a subsequent uninstall.
+====
```

Comment:
Github wouldn't let me comment on the correct line, but on line 63:
> . (Optional) Decouple the data and configuration directories from the binary files by setting the environment variable `NEO4J_CONF` and `server.directories.data` to point to the desired locations.

This step is no longer needed.